src/Game.cpp

```diff
@@ -1,5 +1,39 @@
 #include "Game.hpp"
+#include<vector>
+#include<iostream>
+#include<algorithm>
+#include<ostream>
+#include<string>
+#include<iterator>
 
 Game::Game() {}
 
 void Game::addFrames(std::string const & frames) {}
+
+int Game::conversion_to_int(std::string input)
+{
+    if (input == "-")
+        return 0;
+    else if (input == "X")
+        return 10;
+    else if (input == "1")
```

Comment:
W ramach refaktoringu dobrze by było to zmienić. Ponieważ to jest string to nie możemy zastosować porównań >= i <=, ale możemy wyciągnąć z niego pojedynczy element do zmiennej typu char, który już tak porównamy.
```cpp
char firstCharacter = input[0];
// ...
if (firstCharacter >= '1' and firstCharacter <= '9')
    return std::atoi(firstCharacter);
```